engine/src/main/java/com/datatorrent/stram/engine/Stream.java

```diff
@@ -32,7 +33,7 @@
 /*
  * Provides basic interface for a stream object. Stram, StramChild work via this interface
  */
-public interface Stream extends Component<StreamContext>, ActivationListener<StreamContext>, Sink<Object>
+public interface Stream extends Component<StreamContext>, ActivationListener<StreamContext>, ControlTupleEnabledSink<Object>
```

Comment:
Does the stream need to be control tuple aware? It is still a single stream isn't it, it is the nodes that are de-multiplexing.